CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
+
+### Added
+
+- Add isEVMAccount helper ([#297](https://github.com/MetaMask/keyring-api/pull/297))
+- Add bip122:p2wpkh account support ([#294](https://github.com/MetaMask/keyring-api/pull/294))
```

Comment:
Also, we have the PR that splitted the account types, so IDK if we should mention this one here, or has it as another "bullet item", WDYT?